aips/aip-x.md

```diff
@@ -0,0 +1,103 @@
+- mBackward Compatibility: The existing account model is fully preserved, and this update works seamlessly with existing accounts and applications.
+
+
+Limitations:
+- No Advanced Features Without Resource Creation: Stateless Accounts cannot perform actions like key rotation or use capability offers until the account resource is created.
+- Default Behavior: Stateless Accounts will use default values, such as auth_key = account_address, until the account resource is created.
+
+## Impact 
+- Developer Experience:
+  - Developers should ensure that their dApps handle accounts that may not have an account resource but can still function with default values, particularly for transactions where the auth_key defaults to the account address.
+  - No major code changes are necessary unless applications rely on fields from the account resource (e.g., sequence number, authentication key changes).
+- Account Resource Creation:
+  - When the user performs an action that requires non-default behavior (e.g., key rotation), the account resource will be created automatically, and gas costs will be incurred at that time.
+- Existing Accounts:
+  - Existing accounts with resources already created will continue to function as they do today. This update mainly impacts new accounts or accounts that have not yet performed actions requiring the account resource.
+  
+
+## Alternative Solutions
+The primary alternative to Stateless Accounts is the existing account model, where account resources are always created upfront. Another alternative is the LiteAccount model, which reduces storage overhead by introducing a new account module which needs more engineering work and migration overhead.
+
+Stateless Accounts provide the most gas-efficient solution by avoiding resource creation unless necessary.
+
+## Specification and Implementation Details
+Stateless Account Behavior
+Under the Stateless Account model, accounts that do not have an account resource are treated as valid by default. Specifically:
+
+- Default `auth_key` Behavior: If an account resource does not exist, the account’s auth_key defaults to the account address. This allows the account to sign transactions without needing a full account resource.
+- Resource Creation: If an action requires the account resource (such as rotating the auth_key), the resource will be created at that time. Once created, the account will no longer rely on default values.
+The Move VM will handle these default behaviors in the prologue:
+
+```move
+if !exists<account::Account>(account_address) {
+    auth_key = account_address;
+}
+```
+
+### Sequence Number Considerations
+While the sequence number is currently required for transaction ordering, the sequence number is not created unless the account resource exists. The future introduction of orderless_txn will remove the sequence number requirement entirely, enabling fully stateless transactions.
```

Comment:
add that, call to get_sequence_number will return 0 for stateless accounts that only used orderless_txns thus far.